org.thecrossingchurch.CustomJobs/packages.config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="EntityFramework" version="6.1.3" targetFramework="net472" />
+  <package id="Newtonsoft.Json" version="13.0.1" targetFramework="net472" />
+  <package id="RestSharp" version="105.2.3" targetFramework="net472" requireReinstallation="true" />
+  <package id="SquaredUp.HubSpot.NET" version="0.6.17" targetFramework="net472" requireReinstallation="true" />
+  <package id="System.Diagnostics.DiagnosticSource" version="4.7.1" targetFramework="net472" requireReinstallation="true" />
```

Comment:
The version of `System.Diagnostics.DiagnosticSource` that the HubSpot project is using (v4.7.1) is behind the version that the rest of Rock is using (v5.0.0). Having different versions of the same package can cause issues. Can we update this to v5.0.0?

Same thing for `EntityFramework`. Rock's core code is using v6.4.4, whereas the HubSpot project is using v6.1.3. Can we update this to be in line with core too?